lib/logstash/outputs/email.rb

```diff
@@ -131,7 +107,7 @@ def register
         delivery_method :@via, options
```

Comment:
I know its not part of your refactor, but does this line actually work? I would have thought it should be `@via.to_sym`.